tests/test-performance-lastpostmodified.php

```diff
@@ -20,12 +25,6 @@ public function test__transition_post_status__save_on_publish() {
 		$this->assertEquals( 1, did_action( 'wpcom_vip_bump_lastpostmodified' ) );
 	}
 
-	public function test__transition_post_status__save_on_update() {
-		\wp_transition_post_status( 'publish', 'publish', $this->post );
-
-		$this->assertEquals( 1, did_action( 'wpcom_vip_bump_lastpostmodified' ) );
-	}
```

Comment:
Exactly the same implementation as in `test__transition_post_status__save_on_publish()`